rdflib/exceptions.py

```diff
@@ -27,6 +29,31 @@ def __str__(self):
         return self.msg
 
 
+class ResolutionError(Error):
+    """Resolution error.
+
+    This includes the case when the URI scheme is not supported.
+    """
+
+    def __init__(self, msg):
+        Error.__init__(self, msg)
+        self.msg = msg
+
+    def __str__(self):
+        return self.msg
+
+
+class ResolutionForbiddenError(Error):
+    """Resolution of the given URL is not allowed."""
+
+    def __init__(self, url):
+        Error.__init__(self, self.__doc__)
+        self.url = url
+
+    def __str__(self):
+        return f"Resolution of '{self.url}' is not allowed."
```

Comment:
These should inherit from ValueError also I think so I will change it to do that.